main.py

```diff
@@ -2,6 +2,55 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        )
+    for first_name, last_name in actors:
+        Actor.objects.create(
+            first_name=first_name.strip(),
+            last_name=last_name.strip()
+        )
+
+    Actor.objects.filter(
+        first_name="George",
+        last_name="Klooney"
+    ).update(last_name="Clooney")
+    Actor.objects.filter(
+        first_name="Kianu",
+        last_name="Reaves"
+    ).update(
+        first_name="Keanu",
+        last_name="Reeves"
+    )
+
+    Genre.objects.filter(
+        name="Dramma"
+    ).update(name="Drama")
+
+    Genre.objects.filter(
+        name="Action"
+    ).delete()
+
+    Actor.objects.filter(
+        first_name="Scarlett"
+    ).delete()
+
+    return Actor.objects.filter(
+        last_name="Smith"
+    ).order_by("first_name")
+
+
+genres = ["Western", "Action", "Dramma"]
```

Comment:
i guess this should be moved inside main function because now it is being passed as global